src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,56 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      return onTabSelected(tab);
+    }
+
+    return null;
```

Comment:
You could just `return` in this case.
```suggestion
    return;
```